src/Sulu/Component/Rest/Exception/ConstraintViolationException.php

```diff
@@ -18,36 +18,27 @@ class ConstraintViolationException extends RestException
 {
     public const UNIQUE = 'unique';
 
-    /**
-     * @var string
-     */
-    protected $field;
-
-    /**
-     * @var string
-     */
-    protected $entity;
-
-    /**
-     * @var string
-     */
-    protected $type;
-
     /**
      * @param string $field
+     * @param string $entity
+     * @param string $type
+     * @param int $code
      */
-    public function __construct($field, $entity, $type, $code)
-    {
-        $this->field = $field;
-        $this->entity = $entity;
-        $this->type = $type;
-
+    public function __construct(
+        protected $field,
+        protected $entity,
+        protected $type,
+        protected $code
```

Comment:
```suggestion
       $code
```

Code should just given to the parent